core/domain/suggestion_services.py

```diff
@@ -1826,3 +2020,866 @@ def update_question_suggestion(
+    submitter when a question is reviewed.
+
+    Args:
+        suggestion: Suggestion. The suggestion domain object that is being
+            reviewed.
+    """
+    topic_ids = [
+        topic.topic_id
+        for topic in skill_services.get_all_topic_assignments_for_skill(
+            suggestion.target_id)
+    ]
+
+    for topic_id in topic_ids:
+        question_contribution_stat_model = (
+            suggestion_models.QuestionContributionStatsModel.get(
+                suggestion.author_id, topic_id
+            ))
+
+        if question_contribution_stat_model is None:
+            suggestion_models.QuestionContributionStatsModel.create(
+                contributor_user_id=suggestion.author_id,
+                topic_id=topic_id,
+                submitted_questions_count=1,
+                accepted_questions_count=0,
+                accepted_questions_without_reviewer_edits_count=0,
+                first_contribution_date=_get_date_as_string(
+                    suggestion.last_updated),
+                last_contribution_date=_get_date_as_string(
+                    suggestion.last_updated)
+            )
+            continue
+
+        question_contribution_stat = (
+            _create_question_contribution_stats_from_model(
+                question_contribution_stat_model))
+
+        suggestion_is_accepted = (
+            suggestion.status == suggestion_models.STATUS_ACCEPTED
+        )
+
+        question_contribution_stat.accepted_questions_count += int(
+        suggestion_is_accepted)
+        (
+            question_contribution_stat
+            .accepted_questions_without_reviewer_edits_count
+        ) += int(suggestion.edited_by_reviewer)
+        _update_question_contribution_stats_models(
+            [question_contribution_stat])
+
+
+def update_question_review_stats(
+    suggestion: suggestion_registry.BaseSuggestion
+) -> None:
+    """Creates/updates QuestionReviewStatsModel model for given question
+    reviewer when a question is reviewed.
+
+    Args:
+        suggestion: Suggestion. The suggestion domain object that is being
+            reviewed.
+    """
+    topic_ids = [
+        topic.topic_id
+        for topic in skill_services.get_all_topic_assignments_for_skill(
+            suggestion.target_id)
+    ]
+    suggestion_is_accepted = (
+        suggestion.status == suggestion_models.STATUS_ACCEPTED
+    )
+
+    for topic_id in topic_ids:
+        question_review_stat_model = (
+            suggestion_models.QuestionReviewStatsModel.get(
+                str(suggestion.final_reviewer_id), topic_id
+            ))
+
+        if question_review_stat_model is None:
+            suggestion_models.QuestionReviewStatsModel.create(
+                reviewer_user_id=str(suggestion.final_reviewer_id),
+                topic_id=topic_id,
+                reviewed_questions_count=1,
+                accepted_questions_count=(1 if suggestion_is_accepted else 0),
```

Comment:
Somewhere you use `int(<boolean>)` but here you use `1 if <boolean> else 0`, can you please use one approach everywhere.

Reply:
Made everythong as `1 if <boolean> else 0`